packages/pci-sdk-iframe/src/pages/app/components/Form/Form.tsx

```diff
@@ -17,7 +17,7 @@ export default function Form(props: IFormProps) {
 			>
 				<input
 					style={props.theme.formOTPInput}
-					autoComplete="off"
+					autoComplete="one-time-code"
```

Comment:
Should we call this component now `<OTPForm />` given now it's oriented to OTP only (because of the auto-complete) ? 💭
 
Otherwise, can we create a generic `<Form />` and just pass custom props and use them in both cases?